VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioComponentDetails.cpp

```diff
@@ -96,6 +98,23 @@ bool IsVitruvioComponentSelected(const TArray<TWeakObjectPtr<UObject>>& ObjectsB
 	return false;
 }
 
+void GetAllVitruvioComponents(const TArray<TWeakObjectPtr<UObject>>& ObjectsBeingCustomized, 
+								TArray<UVitruvioComponent*>& OutVitruvioComponents)
+{
+	for (size_t ObjectIndex = 0; ObjectIndex < ObjectsBeingCustomized.Num(); ++ObjectIndex)
```

Comment:
We usually prefer the range based foor loop instead of accessing the elements via their index (if it is not needed)

Eg: `for (const TWeakObjectPtr<UObject>& CurrentObject : ObjectsBeingCustomized)`